docs/dg/dev/best-practices/project-nfr-templates/process-documentation-guidelines.md

```diff
@@ -0,0 +1,118 @@
+
+Given the size and complexity of a large applications, which delivers many features with each release, unexpected errors can have multiple potential resolutions. To ensure that the most effective resolution is chosen with minimal disruption to the functionality of a successfully deployed release, implement the following operation guidelines.
+
+### Main workflows
+To understand the main and critical workflows in the application, operational guidelines should outline the normal behavior of
+important features and workflows at the logical, component, and infrastructure levels.
+
+This forms a general overview of the logic, components, and infrastructure. This overview is necessary for making informed operational decisions and building project-specific operational dashboards, for example—to to identify and highlight project-specific bottlenecks. Also, it's needed for analyzing, answering, and resolving requests.
+
+### Risks, early warnings, and counter actions
+Building a large application is usually coupled with massive application-level logging. In some cases, a critical system issue can be prevented or minimized with timely warning signals. By using regular signals, like logs and metrics, from identified business or technical bottlenecks or risks, the operations team can improve the application's stability. We recommend maintaining a list of such signals and risks in the operational guidelines. This includes recommended actions to take in order to deliver the best mitigation strategy. For more information on these guidelines, see [Operatable feature guidelines](docs/scos/dev/guidelines/operatable-feature-guidelines.html).
+
+### Silent undesired scenario
+Although monitoring systems are used to detect unwanted states, some business functionality may be within acceptable metrics but not desirable under certain conditions. For example, the minimum daily browsing customer count is 10, but 11 customers during Black Friday is not considered normal. To monitor such cases, these scenarios need to be documented for the operations team.
+
+### User guide
+To minimize the impact of resolution efforts and optimize the process, make sure to provide the operations team with the necessary business context. For example, resolving a small number of errors through the Back Office may be more optimal than rolling back an entire release.
+
+### Entity size expectations
+
+The cardinality of entities plays an important role in determining how an application will function in a production environment. To mitigate risks in this area, performance, scaling, manual tests, and reviews can be applied. However, the identified and estimated entity cardinality for the project can serve as a warning to pay extra attention to these entities in order to further reduce risks. For example, a marketing campaign bringing 10,000 active users to the homepage at 17:08 on Friday.
```

Comment:
Is it possible to replace `cardinality`, even if we have to rewrite this a bit?

Reply:
`The entity amount plays an ` ... ?